products/jbrowse-desktop/src/rpc.worker.ts

```diff
@@ -47,6 +48,8 @@ async function getPluginManager() {
   const runtimePlugins = await pluginLoader.load()
   const plugins = [...corePlugins.map(p => ({ plugin: p })), ...runtimePlugins]
   const pluginManager = new PluginManager(plugins.map(P => new P.plugin()))
+  // need to add to the startScreen/util as well
+  pluginManager.addRpcMethod(() => new TextIndexRpcMethod(pluginManager))
```

Comment:
could consider making a little plugin that is desktop-specific and add it to the desktop corePlugins, but that's not a high priority. just a small possible cleanup

Reply:
at least add a comment about what this is here if you keep it

Reply:
removed, and added tiny plugin